src/importexport/musicxml/tests/tst_mxml_io.cpp

```diff
@@ -155,6 +155,7 @@ private slots:
     void hello() { mxmlIoTest("testHello"); }
     void helloReadCompr() { mxmlReadTestCompr("testHello"); }
     void helloReadWriteCompr() { mxmlReadWriteTestCompr("testHello"); }
+    void hiddenStaves() { mxmlIoTest("testHiddenStaves"); }
```

Comment:
@shoogle I don't need to do this for non-show `Parts` case because when you export to MusicXML, it writes the `Part` as hidden if all its staves are also hidden (unlike mscz exports)